src/test/java/net/masterthought/cucumber/generators/integrations/FailuresOverviewPageIntegrationTest.java

```diff
@@ -69,21 +69,4 @@ public void generatePage_onJsonWithoutFailedSteps_generatesProperMessage() {
         SummaryAssertion summary = document.getReport();
         assertThat(summary.getEmptyReportMessage()).isEqualTo("You have no failed scenarios in your Cucumber report");
     }
-
-    @Test
-    public void generatePage_generatesSummary() {
-
```

Comment:
this test checks if for the 'failure overview' page there is only one scenario with expected name - removing this test we loose validation what is really displayed on the page